examples/specs/window_transform_average_rating.vl.json

```diff
@@ -7,7 +7,8 @@
         "values": [
             { "name": "Movie 1", "Rating": 9, "Year": 2016 },
             { "name": "Movie 2", "Rating": 3, "Year": 2016 },
-            { "name": "Movie 3", "Rating": 5, "Year": 2015 }
+            { "name": "Movie 3", "Rating": 5, "Year": 2015 },
+            { "name": "Movie 4", "Rating": 2, "Year": 2015 }
         ]
     },
     "layer": [{
```

Comment:
Not sure why you use `layer` and only have one layer. 

That said, a better plot would be overlaying it with ticks showing `AverageYearRating`.